src/logic/crystallineGiant.ts

```diff
@@ -149,25 +151,32 @@ export class CrystallineGiantWrapper {
 }
 
 export class CrystallineGiant {
-  private state: CrystallineGiantState;
+  private wrapper: CrystallineGiantWrapper;
```

Comment:
Is a `Crystalline Giant` in a `Wrapper` a burrito?

Reply:
One could argue sushi or calzone following the [Cube Rule](https://cuberule.com/). Now between those two? Let's see.

The structure of sushi is defined by having two open ends, if we apply the metaphor to code one can imagine you'd be accessing things inside the wrapper without first going through the wrapper or at least it isn't enforced. 

I would argue it's good practice to adhere to the [Law of Demeter](https://en.m.wikipedia.org/wiki/Law_of_Demeter) when possible.

Therefore, Crystalline Giants wrapper _should_ be a calzone. Reviewing the code again, it seems that the wrapper exposes none of its internals publicly, making it a calzone.